pages/webhosting/reference-content/php-version-overview.mdx

```diff
@@ -7,34 +7,34 @@ content:
-| PHP 4.4         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 5.2         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 5.4         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 5.5         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 5.6         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 7.0         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 7.1         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 7.2         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 7.3         | ✔️                  | ✔️                 | ❌                 | ❌                 |
-| PHP 7.4         | ❌                 | ✔️                 | ✔️                  | ❌                 |
-| PHP 8.0         | ❌                 | ✔️                 | ✔️                  | ❌                 |
-| PHP 8.1         | ❌                 | ✔️                 | ✔️                  | ✔️                  |
-| PHP 8.2         | ❌                 | ❌                | ✔️                  | ✔️                  |
-| PHP 8.3         | ❌                 | ❌                | ✔️                  | ✔️                  |
+| PHP Version     | Dedibox Classic    | Dedibox cPanel    |  Scaleway cPanel    |
+|-----------------|--------------------|-------------------|---------------------|
+| PHP 4.4         | S                  | S                 | NS                 |
+| PHP 5.2         | S                  | S                 | NS                 |
+| PHP 5.4         | S                  | S                 | NS                 |
+| PHP 5.5         | S                  | S                 | NS                 |
+| PHP 5.6         | S                 | S                 | NS                 |
+| PHP 7.0         | S                 | S                 | NS                 |
+| PHP 7.1         | S                 | S                 | NS                 |
+| PHP 7.2         | S                 | S                 | NS                 |
+| PHP 7.3         | S                 | S                 | NS                 |
+| PHP 7.4         | NS                 | S                 | S                  |
+| PHP 8.0         | NS                 | S                 | S                  |
+| PHP 8.1         | NS                 | S                 | S                  |
+| PHP 8.2         | NS                 | NS                | S                  |
+| PHP 8.3         | NS                 | NS                | S                  |
 
 **Key:**
-- ✔️ = PHP version is supported
-- ❌ = PHP version is not supported
+- S = PHP version is supported
+- NS = PHP version is not supported
```

Comment:
Any reason why not use "Yes" or "No"?

Reply:
No particular reason, I can change it to “Y” and “N” but since there’s a legend to explain, I figured I could leave it like that